src/app/models/vehicle-tech-record.model.ts

```diff
@@ -140,4 +140,10 @@ export interface TechRecordModel {
   numberOfSeatbelts?: string;
   seatbeltInstallationApprovalDate?: string;
   departmentalVehicleMarker: boolean;
+  dimensions?: {
```

Comment:
Create a type for this, don't just list it out like this. See here for the api and how it is being sent -> https://github.com/dvsa/cvs-svc-technical-records/blob/develop/%40Types/ITechRecord.d.ts